devices/stm32/stm32f0-30.xml

```diff
@@ -658,7 +658,8 @@
         <signal af="0" driver="tim" instance="3" name="ch2"/>
       </gpio>
       <gpio device-pin="r" port="c" pin="8">
-        <signal af="0" driver="tim" instance="3" name="ch3"/>
+        <signal device-size="c" af="0" driver="tim" instance="3" name="ch3"/>
+        <signal device-size="8" af="1" driver="tim" instance="3" name="ch3"/>
```

Comment:
All these changes with `af="1"` for F0x8 devices are looking a bit disconcerting. The datasheet says the register to set the port C alternate function number does not even exist on size `8` devices, only on size `C` devices and there it is the default `af="0"`. The errata sheet also has nothing. I am wondering if ST is introducing a new mistake here or if they are fixing one …